apps/mdn/utils/percona_toolkit/run.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+${PT_CMD}
```

Comment:
Nit: I'd be more comfortable with something that splits the command from the secret DSN, like:

```
pt-kill --log --busy-time=1h --kill ${PT_DSN}
```

but we can climb that hill if and when we change the command,